### PR TITLE
CP-42186: Rename CHCUpdates mentions and schema into XC based ones

### PR DESCRIPTION
**Pull request by danilo-delbusso**
Don't merge until tags are present in relevant repositories.

### CODE REVIEW

**Comment by danilo-delbusso** [comment]
Blocked because we want to merge it after https://github.com/xenserver/xenadmin/pull/3096

**Review by CitrixChris — APPROVED** [review]

File: scripts/xenadmin-build.sh [hunk 1]
```diff
@@ -225,14 +225,14 @@ msi_checksum=($msi_checksum_with_file_name)
...
+echo "INFO: Generating XCUpdates.xml"
 
 # UPDATE_URL points at the updates XML, we need to point to the MSI
-msi_url="${UPDATES_URL/CHCUpdates.xml/$BRANDING_BRAND_CONSOLE_NO_SPACE.msi}"
+msi_url="${UPDATES_URL/XCUpdates.xml/$BRANDING_BRAND_CONSOLE_NO_SPACE.msi}"
 
 output_xml="<?xml version=\"1.0\" ?>
 <patchdata>
-    <chcversions>
+    <versions>
```
Comment (kc284) [inline scripts/xenadmin-build.sh:235]: The tag is also used in DownloadUpdatesXmlAction.cs.

**Review by kc284 — CHANGES_REQUESTED** [review]
This will also need rebasing.

**Comment by kc284** [comment]
I'm removing the "do not merge" label as the tags referenced by the Jenkinsfile are now in place.

File: scripts/xenadmin-build.sh (hunk 1 above)
Comment (danilo-delbusso) [reply scripts/xenadmin-build.sh:235]: oops! well spotted 😅

**Comment by danilo-delbusso** [comment]
Force push to rebase on `master` as a result of a version number conflict in `Jenkinsfile`

**Review by kc284 — APPROVED** [review]

**Pull request merged**